snap/get_version

```diff
@@ -88,4 +88,10 @@ last_commit_on_main=$(git -C "${SNAPCRAFT_PART_SRC}" merge-base main HEAD)
 last_commit_on_main=$(git -C "${SNAPCRAFT_PART_SRC}" rev-parse --short=7 "${last_commit_on_main}")
 version="${version}.${last_commit_on_main}"
 
+# Check if the version is a valid semantic version.
+if ! semver check "${version}"; then
```

Comment:
This will fail for snap build until we tag the new google branch though as it will have "notag"

Reply:
That's why I added this TODO to the description:

> Tag the commit "Set INITIAL_ALLOWED_USERS_VERSION to 0.2.0-pre1" with 0.2.0-pre1 after this has been merged to main

Once that's done, both the msentraid and google snap should have `0.2.0-pre1`, right?

Reply:
IIRC, we don’t rely on the tag on main anymore but tag each branch manually with msentraid-<version> and google-<version>, no?

Reply:
Oh, I was not aware that's how we do it. I still think we should document our release process.

I added two more TODOs:
> Tag the same commit on the msentraid branch
> Tag the same commit on the google branch

Reply:
You were the one convincing me to ignore the tag on the main branch :D

I think we’ll autotag. I’ll draft something for this.